### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'dbName')  version `0.10.0`

### ISSUE THREAD

**Issue by keonik** [keonik/prisma-erd-generator#75]
```bash
TypeError: Cannot read properties of undefined (reading 'dbName')
```

# schema
```prisma
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}

generator typegraphql {
  provider = "typegraphql-prisma"
}

generator erd {
  provider = "prisma-erd-generator"
}

model Message {
  messageId Int      @id @default(autoincrement()) @map("message_id")
  sender    String
  content   String
  timestamp DateTime @default(now())
  group     Group    @relation(fields: [groupId], references: [groupId], onDelete: Cascade)
  groupId   String
}

model Group {
  groupId  String    @id @default(uuid()) @map("group_id")
  name     String?
  messages Message[]
  app      App       @relation(fields: [appId], references: [appId], onDelete: Cascade)
  appId    String
}

model App {
  appId          String  @id @default(uuid()) @map("app_id")
  name           String  @unique
  messageCounter Int     @default(0) @map("message_counter")
  groups         Group[]
}

```

# dataModelString
```json
{
    "enums": [],
    "models": [
      {
        "name": "Message",
        "dbName": null,
        "fields": [
          {
            "name": "messageId",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": true,
            "isReadOnly": false,
            "type": "Int",
            "hasDefaultValue": true,
            "default": {
              "name": "autoincrement",
              "args": []
            },
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "sender",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "String",
            "hasDefaultValue": false,
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "content",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "String",
            "hasDefaultValue": false,
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "timestamp",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "DateTime",
            "hasDefaultValue": true,
            "default": {
              "name": "now",
              "args": []
            },
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "group",
            "kind": "object",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "Group",
            "hasDefaultValue": false,
            "relationName": "GroupToMessage",
            "relationFromFields": [
              "groupId"
            ],
            "relationToFields": [
              "groupId"
            ],
            "relationOnDelete": "Cascade",
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "groupId",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": true,
            "type": "String",
            "hasDefaultValue": false,
            "isGenerated": false,
            "isUpdatedAt": false
          }
        ],
        "isGenerated": false,
        "primaryKey": null,
        "uniqueFields": [],
        "uniqueIndexes": []
      },
      {
        "name": "Group",
        "dbName": null,
        "fields": [
          {
            "name": "groupId",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": true,
            "isReadOnly": false,
            "type": "String",
            "hasDefaultValue": true,
            "default": {
              "name": "uuid",
              "args": []
            },
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "name",
            "kind": "scalar",
            "isList": false,
            "isRequired": false,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "String",
            "hasDefaultValue": false,
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "messages",
            "kind": "object",
            "isList": true,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "Message",
            "hasDefaultValue": false,
            "relationName": "GroupToMessage",
            "relationFromFields": [],
            "relationToFields": [],
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "app",
            "kind": "object",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "App",
            "hasDefaultValue": false,
            "relationName": "AppToGroup",
            "relationFromFields": [
              "appId"
            ],
            "relationToFields": [
              "appId"
            ],
            "relationOnDelete": "Cascade",
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "appId",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": true,
            "type": "String",
            "hasDefaultValue": false,
            "isGenerated": false,
            "isUpdatedAt": false
          }
        ],
        "isGenerated": false,
        "primaryKey": null,
        "uniqueFields": [],
        "uniqueIndexes": []
      },
      {
        "name": "App",
        "dbName": null,
        "fields": [
          {
            "name": "appId",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": true,
            "isReadOnly": false,
            "type": "String",
            "hasDefaultValue": true,
            "default": {
              "name": "uuid",
              "args": []
            },
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "name",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": true,
            "isId": false,
            "isReadOnly": false,
            "type": "String",
            "hasDefaultValue": false,
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "messageCounter",
            "kind": "scalar",
            "isList": false,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "Int",
            "hasDefaultValue": true,
            "default": 0.0,
            "isGenerated": false,
            "isUpdatedAt": false
          },
          {
            "name": "groups",
            "kind": "object",
            "isList": true,
            "isRequired": true,
            "isUnique": false,
            "isId": false,
            "isReadOnly": false,
            "type": "Group",
            "hasDefaultValue": false,
            "relationName": "AppToGroup",
            "relationFromFields": [],
            "relationToFields": [],
            "isGenerated": false,
            "isUpdatedAt": false
          }
        ],
        "isGenerated": false,
        "primaryKey": null,
        "uniqueFields": [],
        "uniqueIndexes": []
      }
    ]
  }
```

#  mapped models

```json
{mappedModels: [
  {
    name: 'Message',
    dbName: null,
    fields: [ [Object], [Object], [Object], [Object], [Object], [Object] ],
    isGenerated: false,
    primaryKey: null,
    uniqueFields: [],
    uniqueIndexes: []
  },
  {
    name: 'Group',
    dbName: null,
    fields: [ [Object], [Object], [Object], [Object], [Object] ],
    isGenerated: false,
    primaryKey: null,
    uniqueFields: [],
    uniqueIndexes: []
  },
  {
    name: 'App',
    dbName: null,
    fields: [ [Object], [Object], [Object], [Object] ],
    isGenerated: false,
    primaryKey: null,
    uniqueFields: [],
    uniqueIndexes: []
  }
]}
```

mentioned in #62

**Comment by keonik** [comment]
@janpio I'm loving this debug output already 😎 Thank you

**Comment by mmahalwy** [comment]
yep, same here.

**Comment by TMInnovations** [comment]
same here

**Comment by tobkle** [comment]
same problem

**Comment by keonik** [comment]
For now, if you have this issue revert to version `0.9`.

**Comment by keonik** [comment]
Agreed. I'll take it here soon

**Comment by keonik** [comment]
It ended up being an old prisma engine error. The older version I am/was using throws an error for `dml.types` not defaulted to an empty array. So it was pushing an undefined into the `modellikes`. Will be shipped here soon in `0.10.1`
